firewood/src/merkle/node.rs

```diff
@@ -526,39 +286,45 @@ impl Node {
 
 impl Storable for Node {
     fn deserialize<T: CachedStore>(addr: usize, mem: &T) -> Result<Self, ShaleError> {
-        const META_SIZE: usize = 32 + 1 + 1;
+        const META_SIZE: usize = TRIE_HASH_LEN + 1 + 1;
```

Comment:
+1 for replacing 32 with TRIE_HASH_LEN. Any idea what these other +1s are for? Can we const them?

Reply:
I have more PRs coming where this gets cleaned up